indexing-hadoop/src/main/java/io/druid/indexer/JobHelper.java

```diff
@@ -693,13 +707,15 @@ public long push() throws IOException
               try (ZipInputStream in = new ZipInputStream(fileSystem.open(zip, 1 << 13))) {
                 for (ZipEntry entry = in.getNextEntry(); entry != null; entry = in.getNextEntry()) {
                   final String fileName = entry.getName();
-                  try (final OutputStream out = new BufferedOutputStream(
-                      new FileOutputStream(
-                          outDir.getAbsolutePath()
-                          + File.separator
-                          + fileName
-                      ), 1 << 13
-                  )) {
+                  final String outputPath = outDir.getAbsolutePath() + File.separator + fileName;
```

Comment:
`new File(outDir, fileName).getAbsolutePath()` is a more canonical way of doing this.

Reply:
Changed this to `new File(outDir, fileName).getAbsolutePath()`